### PR TITLE
feat: add functions for hitting the `/workers` api endpoints

### PR DESCRIPTION
**Pull request by meinenec**

### CODE REVIEW

File: vela/worker_test.go [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+	if err == nil {
+		t.Errorf("New returned err: %v", err)
+	}
+
+	if resp.StatusCode != http.StatusNotFound {
+		t.Errorf("Worker returned %v, want %v", resp.StatusCode, http.StatusOK)
+	}
+}
+
+func ExampleWorkerService_Get() {
```
[the remark this answers is not in the document]
Comment (meinenec) [reply vela/worker_test.go:243]: these lines are similar, but not duplicative. Even the very first line reference is different.

File: vela/worker_test.go [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+
+	// Set new token in existing client
+	c.Authentication.SetTokenAuth(*auth.Token)
+
+	req := library.Worker{
+		Active: Bool(false),
+	}
+
+	// Update the worker in the server
+	repo, resp, err := c.Worker.Update("worker_1", &req)
```
Comment (kneal) [inline vela/worker_test.go:368]: Believe this should be `repo` -> `worker`

File: vela/worker_test.go [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Printf("Received response code %d, for repo %+v", resp.StatusCode, repo)
```
Comment (kneal) [inline vela/worker_test.go:373]: Believe this should be `repo` -> `worker`

**Review by kneal — CHANGES_REQUESTED** [review]

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Pull request merged**